### PR TITLE
Remove redundant exception

### LINKED ISSUE #8737: [Bug] throws ModuleStartException is redundant
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

In `org.apache.skywalking.oap.server.analyzer.provider.meter.process.MeterProcessorTest#testProcess` 
I think throws ModuleStartException is redundant ,Because I do not find method to throw this exception

### What you expected to happen

do not throw ModuleStartException

### How to reproduce

title

### Anything else

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Oliverwqcwrw**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #8737 .
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: oap-server/analyzer/agent-analyzer/src/test/java/org/apache/skywalking/oap/server/analyzer/provider/meter/process/MeterProcessorTest.java [hunk 1]
```diff
@@ -94,7 +94,7 @@ public void setup() throws StorageException, ModuleStartException {
     }
 
     @Test
-    public void testProcess() throws ModuleStartException {
```
Comment (wu-sheng) [inline oap-server/analyzer/agent-analyzer/src/test/java/org/apache/skywalking/oap/server/analyzer/provider/meter/process/MeterProcessorTest.java:97]: Iam confused, why does an exception statement matter for a UT?

Comment (Oliverwqcwrw) [reply oap-server/analyzer/agent-analyzer/src/test/java/org/apache/skywalking/oap/server/analyzer/provider/meter/process/MeterProcessorTest.java:97]: what does the UT mean?

Comment (wu-sheng) [reply oap-server/analyzer/agent-analyzer/src/test/java/org/apache/skywalking/oap/server/analyzer/provider/meter/process/MeterProcessorTest.java:97]: Unit test. I mean a method with exception should not break anything.

I want to know what do you want to fix.

Comment (Oliverwqcwrw) [reply oap-server/analyzer/agent-analyzer/src/test/java/org/apache/skywalking/oap/server/analyzer/provider/meter/process/MeterProcessorTest.java:97]: I just think the method does not have `throw ModuleStartException` possibility,so I think we can remove the `throws ModuleStartException` let code more clear

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**